components/layout_2020/geom.rs

```diff
@@ -624,7 +624,7 @@ impl ToLogicalWithContainingBlock<LogicalRect<Au>> for PhysicalRect<Au> {
 
 /// The possible values accepted by the sizing properties.
 /// <https://drafts.csswg.org/css-sizing/#sizing-properties>
-#[derive(Clone)]
+#[derive(Clone, PartialEq)]
```

Comment:
It probably makes sense to add a `Copy` implementation to this type if `T` always implements `Copy` as well.

Reply:
OK, then I don't need to `clone()` for `AbsoluteAxisSolver::computed_size`.